src/main/scala/org/broadinstitute/dsde/firecloud/service/TSVFileSupport.scala

```diff
@@ -155,7 +155,8 @@ trait TSVFileSupport {
         // to AttributeString.
         case Success(doubleValue) if !Double.NegativeInfinity.equals(doubleValue)
           && !Double.PositiveInfinity.equals(doubleValue)
-          && !Double.NaN.equals(doubleValue) =>
+          && !Double.NaN.equals(doubleValue)
+        && !matchesLiteral(value) =>
           AttributeNumber(doubleValue)
```

Comment:
I wondered if you could replace the entire `matchesLiteral` method with a library call. We already have apache-commons lang3, so this could be:
```suggestion
          && !Double.NaN.equals(doubleValue)
          && NumberUtils.isParsable(value) =>
            AttributeNumber(doubleValue)
```

buuuuuut this causes one other unit test to fail, on input of "42.": https://github.com/broadinstitute/firecloud-orchestration/blob/ad08ecd794b5a0824e7be3937b9eae55d9921804/src/test/scala/org/broadinstitute/dsde/firecloud/service/TSVFileSupportSpec.scala#L92 … which I think might be a bad test anyway? Do we really want "42." to be interpreted as a number?

Reply:
`NumberUtils.isParsable` also fails on the min and max double values (e.g. 4.9E-324) so it's not looking promising